docs/managed-datahub/subscription-and-notification.md

```diff
@@ -0,0 +1,130 @@
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# Subscriptions & Notifications
+
+<FeatureAvailability saasOnly />
+
+DataHub's Subscriptions and Notifications feature gives you real-time change alerts on data assets of your choice.
```

Comment:
not 100% sure how to message these - technically they aren't available in OSS datahub, but we also use "datahub" as a catch-all term for the overall platform so it might be ok

```suggestion
Acryl Cloud's Subscriptions and Notifications feature gives you real-time change alerts on data assets of your choice.
```